astropy/time/core.py

```diff
@@ -30,7 +30,6 @@
                       TimeJD, TimeUnique, TimeAstropyTime, TimeDatetime)
 # Import TimeFromEpoch to avoid breaking code that followed the old example of
 # making a custom timescale in the documentation.
-from .formats import TimeFromEpoch
```

Comment:
Since you were the one who suggested to keep it, I fear so...  And indeed, `TimeFromEpoch` is imported from `astropy.time.core` in http://docs.astropy.org/en/v1.0.8/time/index.html#writing-a-custom-format